### PR TITLE
servlets

### PR DESCRIPTION
**Pull request by chen678**
Print hard-coded "Hello Chen" message when doGet() is called.

### CODE REVIEW

**Comment by chen678** [comment]
Only the last [commit](https://github.com/chen678/google-software-product-sprint/pull/2/commits/82926554100ad2deabbdbad51c5ae6c57dc0815e) is related to week 2. I created a new branch from branch Chen, which contains the week 1's commits. So this pr contains these commints as well...

File: portfolio/src/main/webapp/WEB-INF/appengine-web.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-  <application>YOUR_PROJECT_ID_HERE</application>
+  <application>cxie-sps-spring20
+</application>
```
Comment (kmontgomery) [inline portfolio/src/main/webapp/WEB-INF/appengine-web.xml]: The spacing looks off on this line.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,83 @@
...
+        <a id ="about">
+            <h1>
+                About me
+            </h1>
+
+            <p>
+                I am from Changchun, a town in the northeast of China. 
+                It is super cold during the winter so I love a tropical weather over it :) 
+                <br />
+                In my life, I love to hang out in the supermarket because the ocean of groceries bring me satisfaction ( ´ ▽ ` )ﾉ
```
Comment (kmontgomery) [inline portfolio/src/main/webapp/index.html:68]: I also love supermarkets! :)

File: portfolio/src/main/webapp/js/script.js [hunk 3]
```diff
@@ -15,14 +15,12 @@
...
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function myFunction() {
```
Comment (kmontgomery) [inline portfolio/src/main/webapp/js/script.js]: Can you add a more descriptive name to this function?

Comment (kmontgomery) [reply portfolio/src/main/webapp/js/script.js]: Can you also add a docstring describing what the method does?

**Review by kmontgomery — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/WEB-INF/appengine-web.xml (hunk 1 above)
Comment (chen678) [reply portfolio/src/main/webapp/WEB-INF/appengine-web.xml]: updated

File: portfolio/src/main/webapp/js/script.js (hunk 3 above)
Comment (chen678) [reply portfolio/src/main/webapp/js/script.js]: updated.

**Comment by chen678** [comment]
addressed comments. Notice that these are mainly related to week 1's work.

**Pull request merged**